assets/admin_themes/Sea_Green/sidebar/sidebar.scss

```
@@ -27,3 +27,22 @@
+        a:hover .survey-settings-panel-item {
+            background-color: $g-200;
+        }
+        a:focus-visible, a:active {
+            outline: none;
+        }
+        a:focus-visible .survey-settings-panel-item,
+        a:active .survey-settings-panel-item{
+            color: $g-800;
+            outline: none;
+            @include inset-border(2px, $grape90);
+            background-color: $white;
```

Stylelint (17.11.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)